tests/Command/RunCommandTest.php

```diff
@@ -210,8 +209,10 @@ public function testExecuteSuccessWithLocalInputOutputInResult(): void
     {
         ['newJobFactory' => $newJobFactory, 'client' => $client] = $this->getJobFactoryAndClient();
 
-        $tableIds = $this->initTestDataTables();
-        $tableId = reset($tableIds);
+        $bucketId = $this->recreateTestBucket();
+        $tableId1 = $this->createTestTable($bucketId, 'someTable', ['a', 'b']);
+        $tableId2 = $this->createTestTable($bucketId, 'someTableNumeric', ['0', '1']);
```

Comment:
Toto je asi jediná obava, kterou bych měl - jestli se ty numerický sloupce někde nepřečíslují - jestli se z 1,2 nestane 0,1 nebo jestli se z 1,0 nestane 0,1. Tak kdyby tě napadlo jak to elegantně ověřit, tak by to bylo super. anebo jestli ti přijde, že je to vyloučené, tak asi není potřeba.

Reply:
No můžu to předělat do dataprovideru a mít víc možností a validovat i data tabulce :)

Reply:
Upravím test a budu validovat i data v tabulce. Našel jsem překlep v jednom asertu tak to stejně musím fixnout :)

Reply:
upravil jsi test tak, aby failoval? :trollface:

Reply:
Upravil jsem jeden test tak aby ostatni failovaly. Uz jsem poslal PR s opravenejma velikostma bytes, tak ted uz to projde.